packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -685,15 +691,15 @@ export const activate = async (extensionContext: vscode.ExtensionContext) => {
       .getConfiguration(SFDX_CORE_CONFIGURATION_NAME)
       .get<boolean>(ENABLE_SOBJECT_REFRESH_ON_STARTUP, false);
 
-    if (sobjectRefreshStartup) {
-      initSObjectDefinitions(
-        vscode.workspace.workspaceFolders[0].uri.fsPath
-      ).catch(e => telemetryService.sendException(e.name, e.message));
-    } else {
-      checkSObjectsAndRefresh(
-        vscode.workspace.workspaceFolders[0].uri.fsPath
-      ).catch(e => telemetryService.sendException(e.name, e.message));
-    }
+    initSObjectDefinitions(
+      vscode.workspace.workspaceFolders[0].uri.fsPath,
+      sobjectRefreshStartup
+    ).catch(e =>
+      telemetryService.sendException(
+        e.name,
+        `Error: ${e.message} with sobjectRefreshStartup = ${sobjectRefreshStartup}`
+      )
```

Comment:
Why not do this error better like you did the other one.  This makes it easy to find these errors in telemetry. Maybe
```
telemetryService.sendException(
        'initSObjectDefinitionsError', // should be a constant
        `Error: name = ${e.name} message = ${e.message} with sobjectRefreshStartup = ${sobjectRefreshStartup}`
      )
```

Reply:
That was the idea but I missed it. Thanks!